src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -233,32 +241,109 @@ def _construct_tsv_content(self):
         output = StringIO()
         writer = csv.writer(output, dialect='excel-tab')
 
-        writer.writerow(list(self.manifest_entries['bundles'].keys()) +
-                        list(self.manifest_entries['contents.files'].keys()))
+        writer.writerow(
```

Comment:
For the time being, don't format code that you wouldn't otherwise touch. Only people familiar with style conventions should do that and only if they keep those cosmetic changes in a separate commit.

Reply:
Reverted, sorry was an accident.